api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb

```diff
@@ -9,6 +9,6 @@ angular.module('mnoEnterprise.configuration', [])
     available_locales: <%= I18n.available_locales.to_json %>
   })
   .constant('PRICING_CONFIG', <%= Settings.pricing.to_json %>)
+  .constant('DOCK_CONFIG', <%= Settings.dock.to_json %>)
   .constant('GOOGLE_TAG_CONTAINER_ID', <%= MnoEnterprise.google_tag_container.to_json %>)
-  .constant('APP_NAME', <%= MnoEnterprise.app_name.to_json %>)
-
+  .constant('APP_NAME', <%= MnoEnterprise.app_name.to_json %>)
```

Comment:
@hedudelgado do not remove newline at end of files. You need to update your editor config. See:
- https://maestrano.atlassian.net/wiki/display/EN/3+-+Development+Environment+Setup#id-3-DevelopmentEnvironmentSetup-Editorconfiguration
- https://robots.thoughtbot.com/no-newline-at-end-of-file